cmd/fyne/version_test.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParsePkgString(t *testing.T) {
+	type test struct {
+		in string // Test input
+		es string // Expected output
+		ee bool   // Expecting error?
```

Comment:
expectedError would imply an error type, perhaps shouldError is closer to the truth for a bool?